server/Server.js

```diff
@@ -1,15 +1,27 @@
 const express = require('express');
+const bodyParser = require('body-parser');
 const db = require('../db/dbConnection.js');
 
 const app = express();
 const port = 3000;
 
 // app.get('/', (req, res) => res.send('Hello World!'))
 
+app.use(bodyParser.json());
```

Comment:
app.use(express.json()) will achieve the same thing without having to require body parser, not sure about urlencoded, would have to consult docs